code/modules/mob/living/simple_animal/hostile/mining_mobs.dm

```diff
@@ -455,8 +455,8 @@
 		if(istype(target, /obj/item/clothing/suit/space/hardsuit/mining) || istype(target, /obj/item/clothing/head/helmet/space/hardsuit/mining))
 			var/obj/item/clothing/C = target
 			var/list/current_armor = C.armor
-			if(current_armor.["melee"] < 80)
-				current_armor.["melee"] = min(current_armor.["melee"] + 10, 80)
+			if(current_armor.["melee"] < 50)
+				current_armor.["melee"] = min(current_armor.["melee"] + 10, 50)
```

Comment:
Yes, 80 is a lot of armor, but it's just a melee value. It's a good protection from toolbox, but the toolboxing assistants are not real killers. Traitors with lasers and revolvers are.